src/open_dread_rando/door_locks/door_patcher.py

```diff
@@ -239,11 +246,12 @@ def door_actor_to_type(self, door: Container, scenario: str) -> DoorType:
         else:
             raise ValueError(f"Door {door.sName} in scenario {scenario} is not a patchable door!")
 
-    def patch_door(self, door_ref: dict, door_type: str):
+    def patch_door(self, door_ref: DefActorReferenceWithLayer, door_type: str) -> None:
         """
         Patches a door given a reference.
 
-        @param door: A dictionary representing a requested door patch.
+        @param door_ref: A dictionary representing a requested door patch.
+        @param door_type:
```

Comment:
Add a description to the parameter otherwise the docstring looks like only half done.

Reply:
I fix the docstring's broken link and you go and ask for me to fix the do string >:(